main.py

```diff
@@ -51,10 +53,22 @@ def end_session(self, session_token: str) -> None:
 
     def get_customer(self, account_number: str) -> Union[Customer, None]:
         for customer in self.customers:
-            if customer.account.account_number == account_number:
-                return customer
+            for account in customer.accounts:
```

Comment:
iterating through a list of all customers in a bank system is not a good idea, it should be implemented by using datastructure that leverages quick lookups